libs/shared/console-shared/src/lib/layout-logs/layout-logs.tsx

```diff
@@ -259,6 +268,19 @@ export function LayoutLogs({
           )}
         </>
       )}
+      {pauseLogs && newMessagesAvailable && (
+        <Button
+          className="absolute bottom-5 left-1/2 flex w-72 -translate-x-1/2 items-center justify-center text-sm"
+          variant="solid"
+          radius="full"
+          size="md"
+          type="button"
+          onClick={() => setPauseLogs?.(false)}
+        >
+          New logs&nbsp;
```

Comment:
Remove this character and using margin

Reply:
Fixed https://github.com/Qovery/console/pull/1587/commits/11c12ebdcf1b2a432051375fdd4ad50d0154f7ff